Zend/zend_fibers.c

```diff
@@ -32,6 +32,10 @@
 # include <valgrind/valgrind.h>
 #endif
 
+#ifdef ZEND_FIBER_UCONTEXT
+# include <ucontext.h>
```

Comment:
This could be moved below to line 76

Reply:
No, it's not possible. Line 76 is in `struct { }`.